modules/1_bastion/bastion.tf

```diff
@@ -162,9 +169,12 @@ EOF
             "sudo yum install -y wget jq git net-tools vim python3 tar"
         ]
     }
+
+# On RHEL 8 don't install ansible using pip but from the ansible repo
     provisioner "remote-exec" {
         inline = [
-            "pip3 install ansible -q"
+            "sudo subscription-manager repos --enable ${var.ansible_repo}",
```

Comment:
Please add a default value to `ansible_repo` in variables.tf, otherwise this command may fail or pause for prompt if not set.